### DISCUSSION TITLE
Should categories with only one level have the choose parent button?

### DISCUSSION THREAD

**Discussion by RyanRoberts** [craftcms/cms#13097] [Q&A] [answered]
In Craft 4.4.7.1 I have categories with only one level, no nesting, but in the category edit page there's a button to select a parent: https://media.yep.digital/8l3dmttH If I create a category from the "choose parent" fly-out panel it's created but disappears in that view as it can't  be selected.

Is this correct or a UI bug?

[edit] These are categories not a structure or entrified category.

**Comment by brandonkelly** [comment, marked as answer]
Yep that was a bug. Just fixed for the next release (5e3762b9b2ae38056e01f0999f442fde4ffbafa6).

**Reply by brandonkelly** [reply]
Craft 4.4.8 is out with that fix! Thanks for reporting.
